snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DefinitionSemanticIT.java

```diff
@@ -41,6 +48,31 @@ public static void shutdown() {
         PrimitiveData.stop();
     }
 
+    @Test
+    public void testDefinitionSemantics() throws IOException {
+        String sourceFilePath = System.getProperty("user.home") + "/Solor/SnomedCT_InternationalRF2_PRODUCTION_20240201T120000Z/Full/Terminology/sct2_TextDefinition_Full-en_INT_20240201.txt";
```

Comment:
The only comment I have is that we make this consistent with the file path and distribution from the other description IT, which would be:

        String sourceFilePath = System.getProperty("user.home") + "/data/SnomedCT_InternationalRF2_PRODUCTION_20241001T120000Z/Full/Terminology/sct2_TextDefinition_Full-en_INT_20241001.txt";